packages/flame/lib/src/game/game.dart

```diff
@@ -250,3 +255,20 @@ class ActiveOverlaysNotifier extends ChangeNotifier {
   /// Returns if the given [overlayName] is active
   bool isActive(String overlayName) => _activeOverlays.contains(overlayName);
 }
+
+/// A [ChangeNotifier] used to control the mouse cursor of the GameWidget which is running this game
+class GameMouseCursorNotifier extends ChangeNotifier {
```

Comment:
Isn't this exactly the same as `ValueNotifier<MouseCursor>`?

Reply:
You are correct, totally forgot about this helper class.